common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -350,7 +351,7 @@ def get_html(self):
             'cdn_eval': cdn_eval,
             'cdn_exp_group': cdn_exp_group,
             'id': self.location.html_id(),
-            'display_name': self.display_name_with_default_escaped,
+            'display_name': self.display_name_with_default,
```

Comment:
@pomegranited: Did you review where this gets used and check to see it is properly escaped?  Typically when escaping gets dropped in one place it needs to be added in another.